ckcp/openshift-operators/openshift-gitops/kustomization.yaml

```diff
@@ -0,0 +1,4 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+- openshift-gitops
```

Comment:
the resource file should be "openshift-gitops.yaml"

Reply:
thanks, done